pkg/lib/serverTime.js

```diff
@@ -321,13 +320,131 @@ export function ServerTime() {
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
+
+    function set_custom_ntp_timesyncd(servers, enabled) {
+        const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
+                                                    { superuser: "try" });
 
-    self.set_custom_ntp = function (servers, enabled) {
         const text = `# This file is automatically generated by Cockpit\n\n[Time]\n${enabled ? "" : "#"}NTP=${servers.join(" ")}\n`;
 
         return cockpit.spawn(["mkdir", "-p", "/etc/systemd/timesyncd.conf.d"], { superuser: "try" })
                 .then(() => custom_ntp_config_file.replace(text));
+    }
+
+    function get_custom_ntp_chronyd() {
+        const result = {
+            supported: true,
+        };
```

Comment:
Matter of taste, but I find this a bit confusing and unwieldy. The `if (data)` below could just `return { supported: true, enabled: true, servers: parse_servers(data) }`.

Reply:
Yes, done.